src/transforms.jl

```diff
@@ -36,7 +36,7 @@ end
     CoordinateTransform
 
 A method to transform the coordinates of objects.
-See [https://en.wikipedia.org/wiki/List_of_common_coordinate_transformations]
+See [https://en.wikipedia.org/wiki/List\\_of\\_common\\_coordinate\\_transformations]
 (https://en.wikipedia.org/wiki/List_of_common_coordinate_transformations).
```

Comment:
```suggestion
See <https://en.wikipedia.org/wiki/List_of_common_coordinate_transformations>.
```

Let's try to fix it with this alternative syntax.